crates/cli/benches/benchmark.rs

```diff
@@ -100,9 +100,18 @@ impl Function {
 }
 
 pub fn criterion_benchmark(c: &mut Criterion) {
-    let functions = fs::read_dir(Path::new("benches").join("functions"))
-        .unwrap()
-        .map(|entry| Function::new(&entry.unwrap().path()).unwrap());
+    let functions = vec![
```

Comment:
> The tradeoff here being that every new function that gets added for benchmarking would have to be included here manually.

Yep. We haven't had to add an additional function yet and AFAIK, we have no plans to. But yes, that's the tradeoff.

> Maybe it's worth setting up simple_discount in the same way so that code can more generically iterate through the functions dir.

What did you have in mind by _same way_? We have at least three options:
1. We could move `index.js` to `dist/function.js` in `simple_discount`. Where things get a little inconsistent is, no one should manually update `dist/function.js` in `complex_discount` but they should manually update it in `simple_discount`.
2. We could add a `npm run build` step to `simple_discount` that copies `index.js` to `dist/function.js` (or runs Vite or whatever). But adding a build step seems like overkill but it would keep the two test cases consistent.
3. We could symlink `index.js` to `dist/function.js` in `simple_discount`. I have to actually test that this'll work as expected when cloning from Github. That would keep the workflow consistent between the two cases and avoid needing to add a build step that's just performing a copy.

Reply:
Yeah (2) was what I had in mind. Basically the same build steps as the `complex_discount` to be consistent. 

> We haven't had to add an additional function yet and AFAIK, we have no 
plans to

Given this though, if you think it's overkill for now, then no worries

Reply:
Yeah. I think we should change the approach I'm using here when we start adding more benchmarks as opposed to now just given that it hasn't changed in a while and probably won't change in the short to medium term.